system-tests/__snapshots__/webpack_dev_server_fresh_spec.ts.js

```diff
@@ -616,6 +625,23 @@ When Cypress detects uncaught errors originating from your test code it will aut
 Cypress could not associate this error to any specific test.
 
 We dynamically generated a new test to display this failure.
+  Error: The following error originated from your test code, not from Cypress.
```

Comment:
Why is this error showing up twice in the output?

Reply:
Will that look weird to the user to have it show up twice? Or is this just a system test nuance?

Reply:
I don't believe so since this is our default behavior if we can discover a message. My guess is previously there wasn't a message attached with the error and now there is, so we have something to print